apps/android/app/src/main/java/chat/simplex/app/views/usersettings/NotificationsSettingsView.kt

```diff
@@ -24,8 +25,11 @@ import chat.simplex.app.views.helpers.*
 import kotlinx.coroutines.*
 import kotlin.collections.ArrayList
 
-enum class NotificationsMode(val requiresIgnoringBattery: Boolean) {
-  OFF(false), PERIODIC(false), SERVICE(true), /*INSTANT(false) - for Firebase notifications */;
+enum class NotificationsMode(private val requiresIgnoringBatterySinceSdk: Int) {
+  OFF(Int.MAX_VALUE), PERIODIC(Build.VERSION_CODES.M), SERVICE(Build.VERSION_CODES.S), /*INSTANT(Int.MAX_VALUE) - for Firebase notifications */;
```

Comment:
I thought this value would somehow affect how this is serialised, but it seems like it's not the case?